starter_code/src/movies.js

```diff
@@ -2,19 +2,82 @@
 // Turn duration of the movies from hours to minutes 
 
 
+function giveMeHours(time) {
+  
+  let hours = time.slice(0, time.indexOf("h"))
+  
+  let minutes = time.slice(time.indexOf(" ") + 1, time.indexOf("min"))
+
+  minutes = parseInt(minutes)
+  hours = parseInt(hours)
+
+  return minutes + hours * 60 
+}
+
```

Comment:
You could write the code slightly more concise here. E.g.
let newArray = array.map(movie => {
   movie.duration = giveMeHours(movie.duration);
  return movie;
  },0)